http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -1291,4 +1323,40 @@ public RequestDistribution(HttpClient httpClient, Processor processor)
             });
         }
     }
+
+    // By wrapping HttpClient, we are able to substitute the underlying AuthenticationStore
+    // with a more efficient one.
+    private static class WrappedHttpClient
+            extends HttpClient
+    {
+        private final AuthenticationStore authenticationStore;
+
+        public WrappedHttpClient(ClientSecurityConfig securityConfig, HttpClientTransport transport,
+                SslContextFactory sslContextFactory)
```

Comment:
We don't wrap arguments around like this. Either put them all on the same line or, if the line is too long, put each in a separate line like this:

```
public WrappedHttpClient(
        ClientSecurityConfig securityConfig, 
        HttpClientTransport transport,
        SslContextFactory sslContextFactory)
{
    ...
}
```